build/charts/theia/values.yaml

```diff
@@ -0,0 +1,27 @@
+clickhouse:
+  # -- Container image to use for the ClickHouse Monitor.
+  monitorImage:
+    repository: "projects.registry.vmware.com/antrea/theia-clickhouse-monitor"
+    pullPolicy: "IfNotPresent"
+    tag: "latest"
```

Comment:
I think something like this would make more sense:
```
clickhouse:
    monitor:
        enable: true
        image:
            repository: ...
            ....
```
there may be more customization later on for the monitor and I think we can also offer the option to disable it?

Reply:
Updated.